sql/wsrep_sst.cc

```diff
@@ -1230,6 +1242,8 @@ int wsrep_remove_sst_user(bool initialize_thread) {
                         nullptr,
                         "SET SESSION lock_wait_timeout = 1;",
                         nullptr,
+                        "REVOKE IF EXISTS 'mysql.pxc.sst.role' FROM 'mysql.pxc.sst.user'@'localhost' IGNORE UNKNOWN USER;",
```

Comment:
This is not needed. Please remove. Related to [THIS](https://github.com/percona/percona-xtradb-cluster/pull/1961/files#r1886504693) comment.

Reply:
done